regolith/builders/recentcollabsbuilder.py

```diff
@@ -139,15 +151,18 @@ def get_ppl_inst_info(self, id, months):
                 emp.sort(key=ene_date_key, reverse=True)
         return ppl_names
 
-    def make_csv_and_excel(self, id, months):
-        '''
+    def make_csv_and_excel(self, ppl_names):
+        """
         function to fill in the 'coa_template.xlsx' and make a csv file with the people and institutions
         information, output from self.get_ppl_inst_info(id, months)
-        args: see self.get_ppl_inst_info(id, months) for details of the args
-        output: None, save an excel file "coa_tables.xlsx" and csv 'recent_collaborators.csv'
-        in _build/recent-collabs/
-        '''
-        ppl_names = self.get_ppl_inst_info(id, months)
+        Parameters
+        ----------
+        ppl_names : list
+            list of tuples. each tuple is a row to be added to the csv and the excel files
```

Comment:
This should say what the thing is rather than its type, which is handled above, so ``"the collaborator information for the rows in the csv and excel files"``